### PR TITLE
User interface improvements and bugfix

### PR DESCRIPTION
**Pull request by szok**
- Add NEW_UI_ENABLED settings. If NEW_UI_ENABLED is True new ui will be displayed by default
- Add handler exception to login page
- Add new ConfigService class
- Bugfix: If open record request and record does not exists raised an error
- Add is_admin method to AuthService
- Move component templates to their catalogs

zrzut ekranu 2016-06-22 o 06 53 58

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: ui/static/app/auth/auth.service.ts [hunk 1]
```diff
@@ -58,6 +61,19 @@ export class AuthService {
   getUserId(): number {
     return Number(this.localStorage.get("auth_user_id"));
   }
+
+  isAdmin(): boolean {
+    return (this.localStorage.get("is_admin") === "true") ? true : false;
```
Comment (mkurek) [inline ui/static/app/auth/auth.service.ts:66]: isn't `return this.localStorage.get("is_admin") === "true"` enough?

Comment (szok) [inline ui/static/app/auth/auth.service.ts:66]: Yes, is ok, in localStorage is saved as a string "true" and method is_admin must return a boolean. It was best solution (for me), any suggestion?

**Comment by mkurek** [comment]
:ok:

**Pull request merged**